src/Concordium/Client/Runner/Helper.hs

```diff
@@ -69,3 +78,6 @@ value s =
   case eitherDecodeStrict' s of
     Right v -> v
     Left err -> error ("Error in gRPC output decoding as a json: " ++ err)
+
+value' :: forall a b. (Field.HasField a "value" b) => GRPCResponse a -> GRPCResponse b
+value' = fmap (^. CF.value)
```

Comment:
It is unclear what this is for. The name is certainly not very meaningful.

It would be better to have something like
```haskell
-- |Extract a value from the response. This assumes that the response from a GRPC call has a @value!@ field.
getValue :: forall a b. (Field.HasField a "value" b) => SimpleGetter (GRPCResponse a) (GRPCResponse b)
getValue = to (fmap (^. CF.value))
```